TramsDataApi/DatabaseModels/Concerns/Case/Management/Actions/Decisions/DecisionTypeId.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace TramsDataApi.DatabaseModels.Concerns.Case.Management.Actions.Decisions
+{
+    // reference table, not required by app in normal use, but used to give context to the data
```

Comment:
It took me a minute to understand the purpose of this. If it is just the database table containing the enum values for lookup, I wonder if it should be a class like this or whether it should be relegated nearer to the migrations or the dbcontext so it's clear that it isn't for use. Maybe one to discuss further, but not going to block the PR for this.

Reply:
In previous projects we didn't have migrations, we just had scripts so it was easy. It could be used, but the comment was just to point out that there's an enum so not much point loading that data from the db unless somehow enhanced in the future